### ISSUE TITLE
Some kind of global variable naming issue regarding `datashader` in holoviews

### ISSUE THREAD

**Issue by PeterDSteinberg** [ioam/holoviews#1176]
Try:
```
from holoviews.operation.datashader import datashade
import datashader
```

It will fail on the second command.

Reverse it and still fails on second command:

```
import datashader
from holoviews.operation.datashader import datashade
```
Suggests some confusion regarding the use of identifier `datashader` within `holoviews.operation`.

This problem exists in Jupyter notebook, but if you go into Ipython console you can do the imports in either order.

**Comment by philippjfr** [comment]
Thanks for reopening this here. Could you confirm the Jupyter, IPython and Python versions you are using? I've tried to reproduce with py2 and py3 without luck.

**Comment by PeterDSteinberg** [comment]
Hi @philippjfr 

Python: `2.7.13`
Here's the Jupyter details:
```
jupyter                   1.0.0                    py27_3
jupyter_client            5.0.0                    py27_0
jupyter_console           5.1.0                    py27_0
jupyter_core              4.3.0                    py27_0
```

Ipython:

```
ipykernel                 4.5.2                    py27_0
ipython                   5.3.0                    py27_0
ipython_genutils          0.1.0                    py27_0
ipywidgets                5.2.2                    py27_1
```

I arrived at my current environment by doing:
```
conda create --name ds27 python=2.7
source activate ds27
conda install -c ioam/label/dev holoviews
conda install datashader
cd $C/datashader
python setup.py develop
conda install -c ioam paramnb
conda install -c scitools/label/dev -c conda-forge iris cartopy
conda install xarray
conda install -c conda-forge -c ioam holoviews>=1.6.2 geoviews
cd $C/holoviews && python setup.py install
```

**Comment by philippjfr** [comment]
Perfect thanks, I'll have a stab.

**Comment by philippjfr** [comment]
@PeterDSteinberg I still haven't been able to reproduce this issue. Are you still seeing this?

**Comment by philippjfr** [comment]
I'm going to close this, since I suspect it's caused by this line:

>conda install -c conda-forge -c ioam holoviews>=1.6.2 geoviews

as it is installing holoviews 1.6.2, which does not include the datashader operations. It should therefore automatically be fixed once 1.7 is out.
